docs/platform/visual/index.md

```diff
@@ -11,10 +11,9 @@ The Pentaho Visualization API provides a unified way to visualize data across th
 [Analyzer](http://www.pentaho.com/product/business-visualization-analytics), 
 [PDI](http://www.pentaho.com/product/data-integration), 
 [CDF](http://community.pentaho.com/ctools/cdf/)).
-Essentially, it is a set of abstractions that ensures total isolation between 
-applications, 
-visualizations and 
-configurations (the glue).
+
+Essentially, it is a set of abstractions that ensures isolation between
+applications, visualizations and configurations.
```

Comment:
I don't have such strong feelings about this particular sentence. I still doubt the strict need for having it, but it does not shock me to have it there. I don't think the matter is worthy of further  discussion.